membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -54,13 +54,16 @@ object AttributesFromZuora extends LoggingWithLogstashFields {
     val attributesFromZuoraOrDynamoFallback: Future[(String, Option[Attributes])] = for {
       dynamoAttributes <- attributesFromDynamo
       attributesFromZuora <- attributesFromZuora
```

Comment:
not introduced in this PR but it's a little confusing having the same name in both sides of the arrow here

Reply:
Agreed. I was surprised that even worked.

Yeah, that was an accident and I'm going to change it.